mffpy/xml_files.py

```diff
@@ -802,3 +921,15 @@ def dipoles(self) -> Dict[str, np.ndarray]:
         Parsing dipoles result in broken shape.  Found {[(k, v.shape) for k, v
         in d_arrays.items()]}"""
         return d_arrays
+
+    def get_content(self):
```

Comment:
Also as a potential / future refactor - since the XML files class is nigh on 1000 lines long, with the possibility of adding additional XML files for reading - we should consider breaking the XML file subclasses into their own files in a future revision.